docs/data-guidelines.md

```diff
@@ -12,6 +12,16 @@ If it's helpful to understanding the rule, summarize the rationale. Definitely c
 
 -- END TEMPLATE -->
 
+
+## Adding subfeatures
+
+Subfeatures should only be added to indicate a  _difference_ in browser compatibility versions from the parent feature; they should not be added just to indicate that a particular feature exists.
+
+For example, the `ImageData` interface was initially defined with `data`, `height` and `width` properties.
```

Comment:
This is not actually true ImageData has these values defined in BCD. We should remove them right @queengooborg ?

FWIW I understand the reason for not documenting "to indicate support", but it does make it hard to know whether you  have just omitted something or there is no known compatibility break.

Reply:
It's actually the opposite: all subfeatures should be added regardless of whether they match the parent version!  Each subfeature should be added to BCD, even if the data matches its parent, to help with clarity.  Say `ImageData` has recently added a new `format` subfeature -- consumers of BCD wouldn't be able to tell whether or not `format` has been supported if the assumption is that all undefined subfeatures match the parent.

Behavioral differences, on the other hand, are more subjective.  If the spec has changed the behavior of a feature and browsers needed to update, then we should add a subfeature for that behavior change.

There are a number of subfeatures we are missing from BCD at the moment, which I'm hoping to fix in the future!

Reply:
@queengooborg OK. That makes sense. It concerns me that this was not communicated to me. Or the new way of adding bug URLs. 

I'll try an update then.

Reply:
P.S. We could reduce duplication by allowing `"support": mirror` for the whole compat section of a subfeature implemented at the same versions as its parent.